js/fixes/img.lazyload.js

```diff
@@ -34,7 +34,7 @@ function applyImgLoadingFix() {
   Adapt.on('reactElement:preRender', event => {
     if (event.name !== 'img') return;
     const options = event.args[1] = event.args[1] || {};
-    if (options && options.hasOwnProperty('loading')) return;
+    if (options && Object.prototype.hasOwnProperty.call(options, 'loading')) return;
```

Comment:
Could you post a picture of the pointing warning please? It was done like this to confirm to the longing rules previously. 

The alternative is `Object.hasOwn(options, 'loading')` so that it works independently of the properties of the object being tested. `hasOwnProperty` can be easily overridden on the`options` object.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn

Reply:
For some reason I thought you were doing the inverse of what I can see now is perfectly valid. Apologies.

Object.hasOwn is just the never version of Object.prototype.hasOwnProperty.